modules/social_features/social_group/modules/social_group_flexible_group/config/update/social_group_flexible_group_update_130006.yml

```diff
@@ -0,0 +1,11 @@
+__global_actions:
+  import_configs:
+    - group.role.flexible_group-verified
```

Comment:
@rochek03 We need to take into account the case that such a group role can already exist. For example we have such error on few of our sites that has this role already:

<img width="1480" alt="image" src="https://github.com/user-attachments/assets/9ba8f829-d7ec-4058-922c-bf61ff779aaa">

Reply:
Fixed